addons/mail/tests/test_mail_channel.py

```diff
@@ -197,6 +201,27 @@ def test_channel_members(self):
         self.assertEqual(channel.message_partner_ids, self.env['res.partner'])
         self.assertEqual(channel.channel_partner_ids, self.env['res.partner'])
 
+    @users('employee')
+    @mute_logger('odoo.addons.mail.models.mail_mail', 'odoo.models.unlink')
+    def test_channel_chat_message_post_should_update_last_activity_time(self):
+        channel = self.env['mail.channel'].browse(self.test_channel.ids)
```

Comment:
you don't need to `browse` again if you already have the channel itself in `self.test_channel`, it will return the same value